SpecFlowProjectMars/StepDefinitions/SkillsTabStepDefinitions.cs

```diff
@@ -0,0 +1,65 @@
+using SpecFlowProjectMars.Pages;
+using SpecFlowProjectMars.Utilities;
+using System;
+using TechTalk.SpecFlow;
+
+namespace SpecFlowProjectMars.StepDefinitions
+{
+    [Binding]
+    public class SkillsTabStepDefinitions : CommonDriver
+    {
+        LoginPage loginPageObj = new LoginPage();
```

Comment:
please implement constructor your object should be moved in one 
Please see 
https://www.knowledgehut.com/tutorials/csharp/csharp-constructors